packages/headless/src/features/facets/facet-search-set/generic/generic-facet-search-actions.ts

```diff
@@ -42,8 +43,10 @@ export const executeFacetSearch = createAsyncThunk<
         state as StateNeededForCategoryFacetSearch
       );
     }
-
-    const response = await searchAPIClient.facetSearch(req);
+    const response = await searchAPIClient.facetSearch({
```

Comment:
It was possible to send falsy date string values because this request's searchContext was unmapped.
Another solution was to change `buildSearchRequest`, but in that case the method would need to return both mappings and request.

Reply:
I think having mapping in `buildSearchRequest` would be aligned with the name of the function, since a valid search request needs to have the mapped values,

Reply:
Okay I'll do that!